bot/src/authorisation.ts

```diff
@@ -0,0 +1,73 @@
+import getSigner from 'auctions-core/src/signer';
+import {
+    authorizeCollateral,
+    authorizeWallet,
+    getCollateralAuthorizationStatus,
+    getWalletAuthorizationStatus,
+} from 'auctions-core/src/authorizations';
+import { ETHEREUM_NETWORK, KEEPER_PREAUTHORIZE, WHITELISTED_COLLATERALS } from './variables';
+import { parseCollateralWhitelist } from './whitelist';
+
+export async function authorizeKeeperWallet(authorizeCallback?: () => void) {
+    const signer = await getSigner(ETHEREUM_NETWORK);
+    const walletAddress = await signer.getAddress();
+    const isWalletAuth = await getWalletAuthorizationStatus(ETHEREUM_NETWORK, walletAddress);
+
+    if (!isWalletAuth) {
```

Comment:
`else` should print "wallet already authorized", right?

Reply:
We did not have this beforehand, but I can add it now!